### DISCUSSION TITLE
Feature Request: Requester Pays AWS s3 support

### DISCUSSION THREAD

**Discussion by gradoj** [duckdb/duckdb#8246] [Ideas]
Please add support for AWS s3 Requester Pays buckets.

https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html

`s3fs.S3FileSystem(anon=False, requester_pays=True)`

**Comment by prmoore77** [comment]
I have added a Pull Request to the httpfs extension to support this here: https://github.com/duckdb/duckdb-httpfs/pull/85
